### DISCUSSION TITLE
eCommerce gallery / Lightbox

### DISCUSSION THREAD

**Discussion by TheoIcelander** [mdbootstrap/TW-Elements#1605] [Share ideas & request features]
I've seen that you have a nice gallery [here](https://tailwind-elements.com/docs/standard/components/gallery/).

But it's missing the option to expand / zoom in on the image & slide between the images.
Maybe something like this:
https://codepen.io/im-vrmedia/pen/BaZvbqW

I think that would be an awesome addition to the components you already have.

**Comment by filipkappa** [comment]
We added this to Tailwind Elements! You can now use the feature:
https://tailwind-elements.com/docs/standard/components/lightbox/
